tutorial/app-init.md

```diff
@@ -4,7 +4,12 @@ Get started by creating a new file: `./app.go`. This file is the heart of your d
 
 In `app.go`, you define what the application does when it receives a transaction. But first, it needs to be able to receive transactions in the correct order. This is the role of the [Tendermint consensus engine](https://github.com/tendermint/tendermint).
 
-Start by importing the necessary dependencies:
+Start by installing and importing the necessary dependencies:
+
+```sh
```

Comment:
I don't think the following two steps were necessary, because they would be call while running `make get_vendor_deps`.

Reply:
Once we reach the end of the tutorial (where we start using deps) I agree :)
But it was a bit weird to follow the tutorial with something that couldn't build, which is why I added that...
Maybe the dep installation should be in the first steps of the tutorial (but that should probably be done in another PR)?

Reply:
I don't think we need this here.

Reply:
Allright I'll remove it then, maybe another change to the tutorial can bring `dep` closer to the introduction.